pennylane/transforms/qcut.py

```diff
@@ -1743,6 +1859,35 @@ def circuit(x):
             :width: 60%
             :target: javascript:void(0);
 
+        If, however, the optimal location of the :class:`~.WireCut` is unknown, we can use
+        :func:`~.find_and_place_cuts` to make attempts in automatically finding such a cut
+        given the device constraints. Using the same circuit as above but with the
+        :class:`~.WireCut` removed, the same (optimal) cut can be recovered with automatic
+        cutting:
+
+        .. code-block:: python
+
+            with qml.tape.QuantumTape() as uncut_tape:
+                qml.RX(0.531, wires=0)
+                qml.RY(0.9, wires=1)
+                qml.RX(0.3, wires=2)
+
+                qml.CZ(wires=[0, 1])
+                qml.RY(-0.4, wires=0)
+
+                qml.CZ(wires=[1, 2])
+
+                qml.expval(qml.grouping.string_to_pauli_word("ZZZ"))
+
+        >>> cut_graph = qml.transforms.qcut.find_and_place_cuts(
+                graph=qml.transforms.qcut.tape_to_graph(uncut_tape),
+                cut_strategy=qml.transforms.qcut.CutStrategy(max_free_wires=2),
+            )
+        >>> print(qml.transforms.qcut.graph_to_tape(cut_graph).draw())
+         0: ──RX(0.531)──╭C──RY(-0.4)──────╭┤ ⟨Z ⊗ Z ⊗ Z⟩
+         1: ──RY(0.9)────╰Z──//────────╭C──├┤ ⟨Z ⊗ Z ⊗ Z⟩
+         2: ──RX(0.3)──────────────────╰Z──╰┤ ⟨Z ⊗ Z ⊗ Z⟩
```

Comment:
```suggestion
        0: ──RX─╭C──RY────┤ ╭<Z@Z@Z>
        1: ──RY─╰Z──//─╭C─┤ ├<Z@Z@Z>
        2: ──RX────────╰Z─┤ ╰<Z@Z@Z>
```
I think the drawing functionality has been recently updated to omit params by default